cowprotocol/accounting/rewards/main_rewards_dashboard_query_2510345.sql

```diff
@@ -4,6 +4,24 @@ block_range as (
     select * from "query_3333356(blockchain='{{blockchain}}',start_time='{{start_time}}',end_time='{{end_time}}')"
 ),
 
+min_auction as (
+    select
+        environment,
+        min(auction_id) as min_auction_id
+    from "query_5270914(blockchain='{{blockchain}}')"
+    where block_deadline >= (select start_block from block_range) and block_deadline <= (select end_block from block_range)
+    group by environment
+),
+
+max_auction as (
+    select
+        environment,
+        max(auction_id) as max_auction_id
+    from "query_5270914(blockchain='{{blockchain}}')"
+    where block_deadline >= (select start_block from block_range) and block_deadline <= (select end_block from block_range)
+    group by environment
+),
+
```

Comment:
I would have expected something like
```suggestion
auction_range as (
    select
        environment,
        min(auction_id) as min_auction_id,
        max(auction_id) as max_auction_id
    from "query_5270914(blockchain='{{blockchain}}')"
    where block_deadline >= (select start_block from block_range) and block_deadline <= (select end_block from block_range)
    group by environment
),

```
and something like
```sql
    inner join auction_range on ad.environment = auction_range.environment
    where ad.auction_id >= auction_range.min_auction_id and ad.auction_id <= auction_range.max_auction_id
```
below.